tests/test_options.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+import re
+
+@pytest.mark.sphinx("html", testroot="simple")
+def test_simple(app, status, warning):
+    app.builder.build_all()
+
+    content = (app.outdir / 'index.html').read_text()
+
+    html = ('<span class="n">Example</span> <span class="n">Domain</span>')
+
+    assert re.search(html, content, re.S)
```

Comment:
This could be changed to use `directives`

Reply:
Directives returns a list of 0 items for some reason.

Reply:
That has to do with the fact that it's looking for `<img>` tags with the `drawio` class, so there would be none in this case...

Reply:
It's probably fine as is.